### DISCUSSION TITLE
Thanks for an easy process

### DISCUSSION THREAD

**Discussion by Dnak-jb** [nod-ai/SHARK#821] [General]
I had to make an account to praise you guys.  I might not be the most well-versed nerd, but I am pretty decent with using a PC.  Trying to get started with local image generation was difficult, to say the least.  Complicating things was I am using Windows and I'm on an ATI card.  Not wanting to duel boot, and after trying a few other methods much more complicated than this. They just were not satisfactory. They were too slow and just had many issues.  Being able to essentially one click and install everything I need and what I want is a godsend.  

You guys might lack a few features that other web UIs use, but that's fine with me. I expect great things out of you folks.  Thanks again!

**Comment by Morichalion** [comment]
I second what this guy said. Been putting random strings into the thing and getting images out. Oddly entertaining.

**Comment by aero909** [comment]
thank you for this amazing solution for amd cards! are you going to make an img2img support in the future?

**Reply by Suplexxer** [reply]
Devs say support coming soonish. Likely within a few weeks.
